diff --git a/sass.dart.js b/sass.dart.js
index d80f073bd8fca6473100a81d3b360289e20881e0..0dc6144e091c66dd5eeb2d4918b6c957e5b9e1c4 100644
--- a/sass.dart.js
+++ b/sass.dart.js
@@ -35,13 +35,13 @@ if (typeof process !== "undefined") {
   self.process = process;
 }
 
-if (typeof __dirname !== "undefined") {
-  self.__dirname = __dirname;
-}
+// if (typeof __dirname !== "undefined") {
+//   self.__dirname = __dirname;
+// }
 
-if (typeof __filename !== "undefined") {
-  self.__filename = __filename;
-}
+// if (typeof __filename !== "undefined") {
+//   self.__filename = __filename;
+// }
 
 if (typeof Buffer !== "undefined") {
   self.Buffer = Buffer;
