diff --git a/sass.dart.js b/sass.dart.js
index cbe14cbde7a754545f8b4264b1e63930cffdb826..dfc96c45de6b7f199b7f91e106bbfd893bc031bb 100644
--- a/sass.dart.js
+++ b/sass.dart.js
@@ -35,13 +35,13 @@ if (typeof process !== "undefined") {
   self.process = process;
 }
 
-if (typeof __dirname !== "undefined") {
-  self.__dirname = __dirname;
-}
+// if (typeof __dirname !== "undefined") {
+//   self.__dirname = __dirname;
+// }
 
-if (typeof __filename !== "undefined") {
-  self.__filename = __filename;
-}
+// if (typeof __filename !== "undefined") {
+//   self.__filename = __filename;
+// }
 
 if (typeof Buffer !== "undefined") {
   self.Buffer = Buffer;
