diff --git a/sass.dart.js b/sass.dart.js
index 4401375d37292b5767b83e4d8e2e1de5338825da..3b9d354f2ffa115e4dace3ccfed69366ac8efa55 100644
--- a/sass.dart.js
+++ b/sass.dart.js
@@ -35,13 +35,13 @@ if (typeof process !== "undefined") {
   self.process = process;
 }
 
-if (typeof __dirname !== "undefined") {
-  self.__dirname = __dirname;
-}
+// if (typeof __dirname !== "undefined") {
+//   self.__dirname = __dirname;
+// }
 
-if (typeof __filename !== "undefined") {
-  self.__filename = __filename;
-}
+// if (typeof __filename !== "undefined") {
+//   self.__filename = __filename;
+// }
 
 if (typeof Buffer !== "undefined") {
   self.Buffer = Buffer;
