diff --git a/sass.dart.js b/sass.dart.js
index aec50edda24eaa529e65a15d649c22c239e03d33..cc1b138ee13c78ab8a156c4a10141b44140be446 100644
--- a/sass.dart.js
+++ b/sass.dart.js
@@ -35,13 +35,13 @@ if (typeof process !== "undefined") {
   self.process = process;
 }
 
-if (typeof __dirname !== "undefined") {
-  self.__dirname = __dirname;
-}
+// if (typeof __dirname !== "undefined") {
+//   self.__dirname = __dirname;
+// }
 
-if (typeof __filename !== "undefined") {
-  self.__filename = __filename;
-}
+// if (typeof __filename !== "undefined") {
+//   self.__filename = __filename;
+// }
 
 if (typeof Buffer !== "undefined") {
   self.Buffer = Buffer;
