diff --git a/dist/resources/helpers/declaration-title.js b/dist/resources/helpers/declaration-title.js
index 0695653c811f67639a504fd56f3cf297545b531d..e96400c24479d1ca959d78584b4a7d6372751a33 100644
--- a/dist/resources/helpers/declaration-title.js
+++ b/dist/resources/helpers/declaration-title.js
@@ -43,9 +43,9 @@ function default_1(theme) {
         }
         md.push(`${this.flags.isRest ? '... ' : ''} **${(0, utils_1.escapeChars)(this.name)}**`);
         if (this instanceof typedoc_1.DeclarationReflection && this.typeParameters) {
-            md.push(`<${this.typeParameters
+            md.push(`&lt;${this.typeParameters
                 .map((typeParameter) => `\`${typeParameter.name}\``)
-                .join(', ')}\\>`);
+                .join(', ')}&gt;`);
         }
         md.push(getType(this));
         if (!(this.type instanceof typedoc_1.LiteralType) &&
diff --git a/dist/resources/helpers/reflection-title.js b/dist/resources/helpers/reflection-title.js
index 41a21f6dfc7c34f1656d227563804ec9b7bd9a3e..bb5a097c51b1d2ed4954a2238ba92d01581a0dab 100644
--- a/dist/resources/helpers/reflection-title.js
+++ b/dist/resources/helpers/reflection-title.js
@@ -42,7 +42,7 @@ function default_1(theme) {
                 const typeParameters = this.model.typeParameters
                     .map((typeParameter) => typeParameter.name)
                     .join(', ');
-                title.push(`<${typeParameters}${shouldEscape ? '\\>' : '>'}`);
+                title.push(`${shouldEscape ? '&lt;' : '<'}${typeParameters}${shouldEscape ? '&gt;' : '>'}`);
             }
         }
         return title.join('');
diff --git a/dist/resources/helpers/signature-title.js b/dist/resources/helpers/signature-title.js
index 42cba7e3b8c47a8b75b88219ee56f33c73e13855..0cea27f7945a3742135f9f2eecb3ac1642061dd4 100644
--- a/dist/resources/helpers/signature-title.js
+++ b/dist/resources/helpers/signature-title.js
@@ -43,9 +43,9 @@ function default_1(theme) {
             md.push(`**${this.name}**`);
         }
         if (this.typeParameters) {
-            md.push(`<${this.typeParameters
+            md.push(`&lt;${this.typeParameters
                 .map((typeParameter) => `\`${typeParameter.name}\``)
-                .join(', ')}\\>`);
+                .join(', ')}&gt;`);
         }
         md.push(`(${getParameters(this.parameters)})`);
         if (this.type && !((_b = this.parent) === null || _b === void 0 ? void 0 : _b.kindOf(typedoc_1.ReflectionKind.Constructor))) {
diff --git a/dist/resources/helpers/type.js b/dist/resources/helpers/type.js
index 26543f2ea14b8e5d46238226ae2b9c6053327c0a..c7d89247579cbef718a341d0d6e5fef14e6028bc 100644
--- a/dist/resources/helpers/type.js
+++ b/dist/resources/helpers/type.js
@@ -113,7 +113,7 @@ function getDeclarationType(model) {
                     ? `= ${(0, utils_1.escapeChars)(obj.defaultValue)}`
                     : ''}`;
             });
-        return `{ ${indexSignature ? indexSignature : ''}${types ? types.join('; ') : ''} }${model.defaultValue && model.defaultValue !== '...'
+        return `\\{ ${indexSignature ? indexSignature : ''}${types ? types.join('; ') : ''} }${model.defaultValue && model.defaultValue !== '...'
             ? `= ${(0, utils_1.escapeChars)(model.defaultValue)}`
             : ''}`;
     }
@@ -122,9 +122,9 @@ function getDeclarationType(model) {
 function getFunctionType(modelSignatures) {
     const functions = modelSignatures.map((fn) => {
         const typeParams = fn.typeParameters
-            ? `<${fn.typeParameters
+            ? `&lt;${fn.typeParameters
                 .map((typeParameter) => typeParameter.name)
-                .join(', ')}\\>`
+                .join(', ')}&gt;`
             : [];
         const params = fn.parameters
             ? fn.parameters.map((param) => {
@@ -136,6 +136,7 @@ function getFunctionType(modelSignatures) {
     });
     return functions.join('');
 }
+
 exports.getFunctionType = getFunctionType;
 function getReferenceType(model, emphasis) {
     var _a;
@@ -150,9 +151,9 @@ function getReferenceType(model, emphasis) {
                 : `\`${model.name}\``);
         }
         if (model.typeArguments && model.typeArguments.length > 0) {
-            reflection.push(`<${model.typeArguments
+            reflection.push(`&lt;${model.typeArguments
                 .map((typeArgument) => Handlebars.helpers.type.call(typeArgument))
-                .join(', ')}\\>`);
+                .join(', ')}&gt;`);
         }
         return reflection.join('');
     }
diff --git a/dist/utils.js b/dist/utils.js
index 9250591a843d085755d8213f0d18d37f769b33fb..3286365ac2b0cc9b9ec93552a3284de59ac0ab73 100644
--- a/dist/utils.js
+++ b/dist/utils.js
@@ -11,7 +11,9 @@ function formatContents(contents) {
 exports.formatContents = formatContents;
 function escapeChars(str) {
     return str
-        .replace(/>/g, '\\>')
+        .replace(/>/g, '&gt;')
+        .replace(/</g, '&lt;')
+        .replace(/{/g, '\\{')
         .replace(/_/g, '\\_')
         .replace(/`/g, '\\`')
         .replace(/\|/g, '\\|');
